src/main.js

```diff
@@ -4,7 +4,14 @@ const utils = require('./utils');
 const Environment = require('./environment');
 const UserAgent = require('./user-agent');
 const requestPromise = require('request-promise');
+const Promise = require('bluebird');
+const PromisePool = require('es6-promise-pool');
+const regeneratorRuntime = require('regenerator-runtime');
+const fs = require('fs');
+require('es6-symbol/implement');
```

Comment:
Node 0.10 is increasingly a rabbit hole. After polyfilling `Promise` with `bluebird`, I was then getting errors about `Symbol`. So I added a `Symbol` polyfill, but then that uncovered more errors:

![image](https://user-images.githubusercontent.com/733696/30431712-7d8f712c-992d-11e7-9e50-f8fe1fc00cef.png)

I think we either need to drop Node 0.10 or I need to find a different `es6-promise-pool`-like solution that doesn't require generators.

Reply:
Let's drop it. Feel free to drop it from the travis yml and I'll do a major bump of this and other repos after this is merged.